### DISCUSSION TITLE
Regular releases

### DISCUSSION THREAD

**Discussion by husayt** [jd-solanki/anu#145] [General]
Hi, first of all thanks for all the effort on the project, I am a strong believer of this direction and a big fan. I have been trying to promote the project to the organisations I work with and one point was coming up is lack of components and no regular releases. I know that there was a stream of updates and new components been added in last months, but many people don't look at commits history and only see latest release. This is why I wanted to suggest to do smaller but regular releases. 

There is enough updates for for multiple releases here and that would give a strong message to those evaluating the project.

Hope this helps. Thanks again.

**Comment by jd-solanki** [comment]
Hi @husayt 

Thanks for raising the concern. Yes, You are right and I do want to release the changes, I was just waiting for a few updates in the `useGroupModel` and wanted to fix `ATooltip`. Anyways, I do feel that this is the right time to make a new release.

Will make a new release today.

**Comment by jd-solanki** [comment]
I just noticed You sponsored the work ❤️ 

Thanks so much so the appreciation 🙏🏻

**Comment by jd-solanki** [comment]
BTW, If you are using it can you please let me know which components you are missing the most so we can prioritize them?

**Reply by husayt** [reply]
Yes, I really miss autocomplete component (which could be derived from AutoSelect) and a [snackbar](https://m2.material.io/components/snackbars#anatomy). There is a pull request #123 for knobs, getting that in would be really useful. 

Also some easy but useful ones : expansion panel, overlay (already exists as part of dialog), tooltip, tabs, range/slider to name the few. 

These would allow for most of usecases and would allow to start a momentum for the library

**Reply by jd-solanki** [reply]
Hi @husayt 

Thanks for your response. For autocomplete, I have to deal with a few hard-to-tackle issues like rendering custom content inside slot (if you noticed we have slot for rendering custom content in options but not in the select items in select component) and improving `useGroupModel` composable.

For snackbar, we just have to plan and prioritize it

I guess you already know we developed tooltip & tabs which will be available in this release (today or tomorrow). For others like expansion panel & range/slider we have to plan and execute them.

Apart from this component thing, we also have to consider a11y and component tests for robustness.

Moreover, I'm also interested in why you want knobs?

**Reply by jd-solanki** [reply]
Hey, @husayt Can we close this discussion?

**Reply by husayt** [reply]
Hi @jd-solanki,

thanks for the release, I will give more feedback as I get to try all the features. Moving forward it will be great to keep releases smaller, but maybe more regular, to make it also easier for users to appreciate all the features.

 Knobs are very useful for infographics and some random graphics without the need to bring another library.
 
 Thanks for all the good effort.
